### PR TITLE
app-cdr/cddetect: EAPI=7 bump

### PR DESCRIPTION
**Pull request by mm1ke**
Hi, 

This is a very simple EAPI=7 bump for app-cdr/cddectect.
Please review.

Signed-off-by: Michael Mair-Keimberger <m.mairkeimberger@gmail.com>

### CODE REVIEW

File: app-cdr/cddetect/cddetect-2.1.ebuild [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+DESCRIPTION="Tool for detecting the type of a CD/DVD without mounting it"
 HOMEPAGE="http://www.bellut.net/projects.html"
 SRC_URI="http://www.bellut.net/files/${P}.tar.gz"
 
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
 
 S=${WORKDIR}
```
Comment (SoapGentoo) [inline app-cdr/cddetect/cddetect-2.1.ebuild]: ...did you really test this ebuild after porting to EAPI 7?

Comment (mm1ke) [reply app-cdr/cddetect/cddetect-2.1.ebuild]: Huh... looks like i really forgot here.. I usually always test my changes of course - sorry for the mistake.

File: app-cdr/cddetect/cddetect-2.1.ebuild [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
 
 src_prepare() {
+	default
 	sed -i -e '1i#include <limits.h>' ${PN}.c || die #337628
```
Comment (SoapGentoo) [inline app-cdr/cddetect/cddetect-2.1.ebuild]: please turn this into a patch and put it in `PATCHES`

Comment (mm1ke) [reply app-cdr/cddetect/cddetect-2.1.ebuild]: Done :)

**Pull request closed without merging**